GsaGH/Components/GraveyardComp/EditElement1d_OBSOLETE.cs

```diff
@@ -242,7 +242,7 @@ protected override void SolveInstance(IGH_DataAccess DA)
         // #### outputs ####
         DA.SetData(0, new GsaElement1dGoo(elem));
         DA.SetData(1, elem.Id);
-        DA.SetData(2, elem.Line);
+        DA.SetData(2, elem.Line.Line);
```

Comment:
sure about that?

Reply:
yes, elem.Line is a LineCurve type and elem.Line.Line is a Line type. The output param is a GH_Line